specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Specify_NoWarn_rule_IDs_fully.cs

```diff
@@ -0,0 +1,29 @@
+namespace Rules.MS_Build.Specify_NoWarn_rule_IDs_fully;
+
+public class Reports
+{
+    [Test]
+    public void IDs_specified_as_ints() => new SpecifyNoWarnRuleIdsFully()
+        .ForInlineCsproj(@"
+<Project Sdk=""Microsoft.NET.Sdk"">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <NoWarn>$(NoWarn);1700;NuGet1700;42</NoWarn>
```

Comment:
Perhaps change the numeric part of `NuGet1700` to be different than the initial `1700` to verify it doesn't trigger a warning due to the number being duplicated.